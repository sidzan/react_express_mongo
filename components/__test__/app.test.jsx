import React from 'react'
import renderer from 'react-test-renderer'
import App from '../Apps/app'

test('This contains List of datas including an input box',()=>{
    const component = renderer.create(
        <div>
            <div key="1">1. Sijan > This is awesome</div>
            <div key="2">1. Beer > This is awesome beer</div>
            <form>
               <input type="text" placeholder="name" name="name"/> 
               <input type="text" placeholder="quot" name="quote"/> 
               <button type="submit">Submit</button>
            </form>
        </div>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    tree.props.submit();

    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
