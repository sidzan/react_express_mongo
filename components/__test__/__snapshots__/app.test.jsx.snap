// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This contains List of datas including an input box 1`] = `
<div>
  <div>
    1. Sijan &gt; This is awesome
  </div>
  <div>
    1. Beer &gt; This is awesome beer
  </div>
  <form>
    <input
      name="name"
      placeholder="name"
      type="text"
    />
    <input
      name="quote"
      placeholder="quot"
      type="text"
    />
    <button
      type="submit"
    >
      Submit
    </button>
  </form>
</div>
`;
